pkg/parser/runtime.go

```diff
@@ -65,11 +64,17 @@ func SetTargetByName(target string, value string, evt *types.Event) bool {
 		case reflect.Struct:
 			tmp := iter.FieldByName(f)
 			if !tmp.IsValid() {
-				log.Debugf("%s IsValid false", f)
+				log.Errorf("%s IsValid false", f)
```

Comment:
if we want to set it to error level, I think we need something more explicit, maybe include `target` in the error message ?